docs/.vitepress/theme/custom.css

```
@@ -1,104 +1,147 @@
+  --vp-c-tip-1: var(--vw-cyan);
+  --vp-c-tip-soft: rgba(1, 205, 254, 0.14);
+}
+
+/* ------------------------------------------------------------------ */
+/* Chrome: navbar, sidebar, docs                                       */
+/* ------------------------------------------------------------------ */
 .VPNavBarTitle .title {
-  font-family: 'VT323', monospace;
-  font-size: 1.5rem;
-  color: var(--usage-green) !important;
+  font-family: 'Orbitron', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Orbitron" (font-family-name-quotes)

(font-family-name-quotes)